test-project/composer.json

```diff
@@ -7,7 +7,12 @@
             "type": "composer",
             "url": "http://localhost:8080",
             "tuf": {
-                "root": "root.json"
+                "root": {
+                    "hashes": {
+                        "sha256": "6cf95b77cedc832c980b81560704bd2fb9ee32ec4c1a73395a029b76715705cc"
```

Comment:
If I remember correctly, the TUF spec locates the root key offline (e.g., literally in a physical safe or something) and suggests rotating it roughly annually. Wouldn't this sha hash change when that happens? And in that case, would this "just stop working" one day for our users until they update the hash from the webpage or whatever they originally got it from?